src/components/molecules/Visualizer/Engine/Cesium/Marker/index.tsx

```diff
@@ -131,9 +131,19 @@ const Marker: React.FC<PrimitiveProps<Property>> = ({ layer }) => {
   );
 
   const pixelOffset = useMemo(() => {
-    const padding = 15;
-    const x = (img?.width && style == "image" ? img.width : pointSize) / 2 + padding;
-    const y = (img?.height && style == "image" ? img.height : pointSize) / 2 + padding;
+    const padding = imageSize ? 15 : 40;
```

Comment:
```suggestion
    const padding = 15;
```

What is your intention in changing the padding? If you don't have any intention, please change it back.

Reply:
The problem we are having right now is the following:
when we upload an image for a marker it will be very small and imageSize(imageScale) will be undefined until we adjust the imageSize(imageScale) manually, that is why I added a margin of 40 in case the imageSize(imageScale) is undefined to avoid overlapping in both cases.

Reply:
this is how the label and image look before manually setting the imageSize(imageScale)
![image](https://user-images.githubusercontent.com/35835415/171604994-09872f98-f312-4906-a449-15bf39b6d881.png)

Reply:
I see. More suggestion: https://github.com/reearth/reearth-web/pull/238#discussion_r888564906